extension/background.js

```diff
@@ -26,13 +26,24 @@ chrome.runtime.onMessage.addListener(
         text = text.filter(function(element) {return element !== undefined;});
         projectID = request.project_id
         filename = request.editingFile;
+        if (request.message == "user_selection"){
```

Comment:
What does user_selection signify?

Reply:
"User selection" represents a user action which user selected a chunk of text and want Chatgpt help. When user doing this action, "user selection" message along with some other informations (selected text, context, line number, time) send to background.js.